scanner/src/main/kotlin/scanners/Licensee.kt

```diff
@@ -135,10 +135,19 @@ class Licensee(name: String, config: ScannerConfiguration) : LocalScanner(name,
     override fun generateSummary(startTime: Instant, endTime: Instant, path: File, result: JsonNode): ScanSummary {
         val matchedFiles = result["matched_files"]
         val licenseFindings = matchedFiles.map {
-            // TODO: Set the location of the finding properly
+            val relativePath = if (path.isFile) {
+                if (path.isAbsolute) {
+                    path.relativeTo(File(".").absoluteFile)
+                } else {
+                    path
+                }
+            } else {
+                File(it["filename"].textValue()).relativeTo(path.absoluteFile)
```

Comment:
Again, why can't this be the default? Also, as the logic is basically the same in all three scanners it would be worth moving this to a helper function, this could also make it easier to understand the logic.